### DISCUSSION TITLE
Point Blank for Securly

### DISCUSSION THREAD

**Discussion by op-max** [3kh0/ext-remover#691] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Point blank for securly isn't working anymore.

**Comment by Aka-but-nice** [comment, marked as answer]
No, it still works. Try visiting chrome://extensions/?id=iheobagjkfklnlikgihanlhcddjoihkg, if it returns to the regular chrome://extensions page, then you may have a different Securly ID.

**Reply by op-max** [reply]
Oh, thanks.

**Reply by synapt0** [reply]
I tried substituting the other Securly ID in the point-blank link and it brought me to the about:blank but after I used the bookmarklet, nothing happened.

**Reply by obkduishinijf** [reply]
@op-max mark as answer

**Comment by Aka-but-nice** [comment]
Also, you are probably on a version above v114 of Chrome, therefore pronouncing Point Blank as patched. In this case, I would recommend using [insecurly](https://disablesecurly.com)

**Reply by AshtonDavies** [reply]
I didn't know Point-Blank got patched back in 115.

**Reply by akabutnicer** [reply]
"in 114" proclaims the exploit was patched, as you said, "in 114". therefore, don't come to me acting like you know everything when you can't read whatsoever, nothing you said implies anything about versions above 114. please gain better reading abilities :pray:

**Reply by AshtonDavies** [reply]
Sorry, because you care so much, you could've clarified the number instead of "above " plus the number. Besides, I don't think it's really patched. 115 is the version bookmarklets are patched for those with devtools disabled and that's probably what you mean for Point-Blank. Believe it or not, there's a theoretical workaround to bookmarklets, so it probably could still work, but that probably doesn't matter in the context of this discussion and I don't care what's going on.

**Reply by akabutnicer** [reply]
114+ clarifies versions 114, and above has point blank patched. therefore, please gain better reading abilities before talking to me, idiot.

**Reply by akabutnicer** [reply]
and of course, i'm not saying bookmarklets are completely **patched**, but with the update itself, excluding fixes with exploits, proclaims point blank as patched.

**Reply by AshtonDavies** [reply]
Yes, and that's not what you said if you go ahead and reread their post. I should probably stop talking to y'all 12 year olds this is stupid of me guess I am an idiot after all.

**Reply by akabutnicer** [reply]
Yeah, 12 year olds for sure :sob::pray::pray:

**Reply by AshtonDavies** [reply]
It's patched for specific users.

**Reply by akabutnicer** [reply]
And? LMAO.

**Reply by AshtonDavies** [reply]
Not necessarily needed to proclaim as patched then, it's to very specific factors that make that.

**Reply by AshtonDavies** [reply]
And for goodness sake, gain at least decent writing abilities before wasting my time on something pathetic that you can always clarify in a friendly way Mr. Aka, but mean.

**Reply by akabutnicer** [reply]
Honestly, its stupid even talking about this. Not sure why this even started in the first place :skull:. Anyway, stay safe (no I'm not saying this because I supposedly lost the argument,  I'm just tired of the back to back conflicting).

**Reply by AshtonDavies** [reply]
Same, thanks for stepping it down, not worth arguing about. 👍
